__init__.py

```diff
@@ -539,7 +539,7 @@ def play_station(self, message=None):
             station = self._extract_station(message.data["utterance"])
 
             if station is not None:
-                self._play_station(station)
+                self._play_station(station[0])
```

Comment:
I would rather see the return value of the _extract_station method unpacked than using the zero index.

`station, _ = self._extract_station(message.data["utterance"])`

Reply:
Per the original PR "Did not unpack the values as _extract_station returns None if no station was found."